src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -15211,6 +15519,19 @@ public partial interface IExpando : System.Reflection.IReflect
 }
 namespace System.Runtime.InteropServices.WindowsRuntime
 {
+    [System.AttributeUsageAttribute((System.AttributeTargets)(5148), Inherited = false)]
+    internal sealed partial class WindowsRuntimeImportAttribute : System.Attribute
+    {
+        internal WindowsRuntimeImportAttribute() { }
+    }
+    [System.Runtime.InteropServices.GuidAttribute("82BA7092-4C88-427D-A7BC-16DD93FEB67E")]
+    [System.Runtime.InteropServices.InterfaceTypeAttribute((System.Runtime.InteropServices.ComInterfaceType)(1))]
+    internal partial interface IRestrictedErrorInfo
+    {
+        void GetErrorDetails(out string description, out int error, out string restrictedDescription, out string capabilitySid);
+        void GetReference(out string reference);
+    }
+#if FEATURE_COMINTEROP
```

Comment:
should we put this around all types that are only used for com interop?
For example, I see VARFLAGS above -- although,, I see that is already exposed in .NET Core (and Mono)